### PR TITLE
Added release notes for mxpccli, clear cache and check for updates te…

### PR DESCRIPTION
**Pull request by Nidhi251289**
…xt and removed references for placeholder app

### CODE REVIEW

**Comment by Adam-Dupaski** [comment]
Hi @MariaShaposhnikova can you prepare this for release for tomorrow? (I have PTO.)

@MarkvanMents please review upon return.

File: content/releasenotes/developer-portal/mendix-for-private-cloud.md [hunk 1]
```diff
@@ -12,6 +12,13 @@ For information on the current status of deployment to Mendix for Private Cloud
 
 ## 2022
 
+### March 4th, 2022
+
+#### mxpc-cli v2.4.1 for Mendix Operator v2.3.0
+
+* We have added an option to clear the local cache and  check for updates to verify that the hash of your downloaded images matches with he remote images.
+* We have also fixed issue with Run installer which was not working as expected
```
Comment (zlogic) [inline content/releasenotes/developer-portal/mendix-for-private-cloud.md:20]: Can you also add a  (Ticket 142583) note here?

File: content/developerportal/deploy/private-cloud-deploy.md [hunk 2]
```diff
@@ -96,49 +126,20 @@ First you need to create an environment:
...
 
-11. You will see your new environment listed. An *in-progress* icon will be shows next to the resource plans until they have been provisioned.
+12. You will see your new environment listed. An *in-progress* icon will be shows next to the resource plans until they have been provisioned.
 
     ![](attachments/private-cloud-deploy/image8.png)
 
-You will not be able to deploy to this environment until it has been fully prepared. This means that all the resource plans have been confirmed and that the placeholder app has been successfully deployed. See [Deploying the Deployment Package](#deploy-package), below, for instructions on how to check that the environment has been created successfully.
+See [Deploying the Deployment Package](#deploy-package), below, for instructions on how to check that the environment has been created successfully.
 
-### 3.3 Creating a Deployment Package {#create-deployment-package}
```
[the remark this answers is not in the document]
Comment (Nidhi251289) [reply content/developerportal/deploy/private-cloud-deploy.md:113]: yes, i have done that already.

File: content/releasenotes/developer-portal/mendix-for-private-cloud.md (hunk 1 above)
Comment (Nidhi251289) [reply content/releasenotes/developer-portal/mendix-for-private-cloud.md:20]: ok

**Review by Nidhi251289 — COMMENTED** [review]
looks good

**Pull request merged**